app-frontend/src/app/components/projects/projectPublishModal/projectPublishModal.module.js

```diff
@@ -162,6 +167,19 @@ class ProjectPublishModalController {
             }
         }
     }
+
+    onCopyClick(e, url, type) {
+        if (url && url.length) {
+            this.copyType = type;
+            angular.element(e.currentTarget).css({
```

Comment:
ask @designmatty if there's a non-javascript way to do this (pure css).
For styles, we want to avoid using js if possible, since it leaves more room for state errors.